### DISCUSSION TITLE
No Space After Icons

### DISCUSSION THREAD

**Discussion by nomasprime** [LazyVim/LazyVim#4908] [Q&A] [answered]
<img width="263" alt="Screenshot 2024-11-29 at 16 57 18" src="https://github.com/user-attachments/assets/769124c0-acf6-435d-b9c7-b55741580f30">

**Comment by dpetka2001** [comment]
I believe that has to do with how the fonts that you use do the rendering or maybe even the terminal you use or even the resolution of your monitor maybe?? This is how it looks for me

![2024-11-29_19-08](https://github.com/user-attachments/assets/553dd606-3446-42f1-a8b2-71c979705b2e)

I use Kitty terminal with fallback `NerdFontsSymbolsOnly`.

**Reply by nomasprime** [reply]
I'm using iTerm and Pragmata Pro, which is patched with Nerd Fonts OOTB. So far, I haven't found an option to change the spacing 😞 If I can't, I'll try something like a fallback. Thanks.

**Reply by folke** [reply, marked as answer]
Make sure you're using the mono variant

**Reply by nomasprime** [reply]
Thanks, @folke, just discovered that. Now I get:

<img width="288" alt="Screenshot 2024-11-29 at 17 29 57" src="https://github.com/user-attachments/assets/43660f60-ec8f-4a76-a092-520734800ef9">

Would prefer larger icons but it'll do.
